components/automate-ui/src/app/modules/infra-proxy/environment-details/environment-details.component.scss

```diff
@@ -167,7 +167,7 @@ img {
   font-size: 12px;
   line-height: 20px;
   letter-spacing: 0.5px;
-  color: #000000;
+  color: $chef-black;
```

Comment:
Author note: Do not use color literals in CSS files; always use the provided named constants.